### ISSUE TITLE
[Bug]: AFP is missing the libevent dependency

### ISSUE THREAD

**Issue by neseleznev** [mbentley/docker-timemachine#183]
### Describe the Bug

The commit https://github.com/mbentley/docker-timemachine/commit/df2e44415f985475af9655f8e9790f7cdd86eaf6 (line 25) brought a bug. `:afp` tag does not start anymore with the error: `netatalk: error while loading shared libraries : libevent-2.0.so.5`

### Expected Behavior

Well, a dependency in the debian image

### Steps to Reproduce

docker run with afp tag

### How You're Launching the Container

```plain
...
```

### Container Logs

```plain
nuff said
```

**Comment by neseleznev** [comment]
```sh
apt-get update -y
apt-get install -y libevent-dev
```

resolves the issue

**Comment by mbentley** [comment]
Thanks for the issue!  Turns out it needed libevent-2.0-5 to supply that file.  It seemed to work here in my test.  There should be a new build on Docker Hub in a minute or two.

**Comment by neseleznev** [comment]
> Turns out it needed libevent-2.0-5

Precisely! It worked in my env as well. Thank you for the fix and your work in general.

_____

My two cents on AFP protocol and what I've chosen are as follows.
I heard that AFP is faster for TimeMachine, but I would've sacrifised any benefits of it in favor of stability, no problem.
The problem is my setup, which is far from unique, I believe!

So, I have a Windows PC, mainly used as a media center and for games. I decided to add a TimeMachine server there. I didn't want to cope with OS switching every time I wanted to play or make a backup, so I decided to host the SMB right there, in Windows host.

1. Windows SMB is not compatible with TM due to the lack of xattr capabilities IIRC
2. Okay, let's go with WSL and the Docker image, I thought. The next snag I hit was that I was unable to block the native SMB port and forward the one from the WSL. You simply can't! It's Windows, you know

Thus, I ended up with the AFP protocol, which finally worked out.
It might sound crazy that to backup **macOS**, I use **Linux** inside **Windows**; I got worst of all worlds 🤣 
But I'm not sure if there's another option to backup TM onto a Windows host
